tools/harness-simulation/harness/Thread_Harness/Sniffer/SimSniffer.py

```diff
@@ -145,53 +141,35 @@ def discoverSniffer(self):
     def startSniffer(self, channelToCapture, captureFileLocation, includeEthernet=False):
         self.channel = channelToCapture
         self._local_pcapng_location = captureFileLocation
-        self._remote_pcap_location = os.path.join(REMOTE_SNIFFER_OUTPUT_PREFIX, self.ipaddr.split('@')[0] + '.pcap')
 
-        self._ssh = paramiko.SSHClient()
-        self._ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        remote_ip = self.ipaddr.split('@')[1]
-        self._ssh.connect(remote_ip, port=REMOTE_PORT, username=REMOTE_USERNAME, password=REMOTE_PASSWORD)
-
-        _, stdout, _ = self._ssh.exec_command(
-            'echo $$ && exec python3 %s -o %s -c %d' %
-            (os.path.join(REMOTE_OT_PATH, 'tools/harness-simulation/posix/sniffer_sim/sniffer.py'),
-             self._remote_pcap_location, self.channel))
-        self._remote_pid = int(stdout.readline())
-
-        self.log('local pcapng location = %s', self._local_pcapng_location)
-        self.log('remote pcap location = %s', self._remote_pcap_location)
-        self.log('remote pid = %d', self._remote_pid)
+        response = self._stub.Start(sniffer_pb2.StartRequest(channel=self.channel))
+        assert response.status == sniffer_pb2.OK
 
         self.is_active = True
 
     @watched
     def stopSniffer(self):
         if not self.is_active:
             return
-        self.is_active = False
 
-        assert self._ssh is not None
-        self._ssh.exec_command('kill -s TERM %d' % self._remote_pid)
-        # Wait to make sure the file is closed
-        time.sleep(3)
+        response = self._stub.Stop(sniffer_pb2.StopRequest())
+        assert response.status == sniffer_pb2.OK
 
-        # Truncate suffix from .pcapng to .pcap
+        # truncate suffix from .pcapng to .pcap
```

Comment:
Why making it lowercase? I can see capitalized comments below.

Reply:
Make the first letter capitalized.